lib/data.js

```diff
@@ -0,0 +1,18 @@
+export const titles = [
+  {
+    date: "22.12.2022",
+    entryTitle: "Entry 1",
+    entry: "Lorem isump 1",
+  },
```

Comment:
add a unique id to each title, you can use that as key while you map:

```suggestion
  {
    id: "Jas8s",
    date: "22.12.2022",
    entryTitle: "Entry 1",
    entry: "Lorem isump 1",
  },
```